doc/source/user_guide/ug_case_settings.rst

```diff
@@ -167,14 +167,19 @@ setup_nml
    "", "``file``", "write diagnostic output to file", ""
    "``diag_file``", "string", "diagnostic output file", "'ice_diag.d'"
    "``dt``", "real", "thermodynamics time step length in seconds", "3600."
-   "``dumpfreq``", "``d``", "write restart every ``dumpfreq_n`` days", "``y``"
+   "``dumpfreq``", "``d``", "write restart every ``dumpfreq_n`` days", "'y','x','x','x','x'"
+   "", "``d1``", "write restart every ``dumpfreq_n`` days only once", ""
```

Comment:
I think this wording might be clearer:

> write restart once, after  ``dumpfreq_n``  days

and similarily for the other frequencies